### PR TITLE
Add 2015-05-01 capabilities swagger

### PR DESCRIPTION
**Pull request by gaagra**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by anuchandy** [comment]
continuous-integration/travis-ci/pr The Travis CI build [failed](https://travis-ci.org/Azure/azure-rest-api-specs/builds/269428052?utm_source=github_status&utm_medium=notification)

@gaagra please fix the linter, sematic and model validations - 

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/269428059
https://travis-ci.org/Azure/azure-rest-api-specs/jobs/269428060
https://travis-ci.org/Azure/azure-rest-api-specs/jobs/269428061

**Comment by anuchandy** [comment]
This is a new swagger targeting API version `2015-05-01` so adding `WaitForARMFeedBack` label.

**Review by anuchandy — CHANGES_REQUESTED** [review]
Please fix the CI failures as pointed. Also provide examples for capabilities operations.

**Comment by anuchandy** [comment]
@gaagra it looks like swagger reference examples which does not exist.

You can repro this locally by running:

```
autorest --validation --azure-validator --message-format=json --input-file=./specification/sql/resource-manager/Microsoft.Sql/2015-05-01/capabilities.json
```

```
oav validate-spec ./specification/sql/resource-manager/Microsoft.Sql/2015-05-01/capabilities.json
oav validate-example ./specification/sql/resource-manager/Microsoft.Sql/2015-05-01/capabilities.json 
```

File: specification/sql/resource-manager/Microsoft.Sql/2015-05-01/capabilities.json [hunk 1]
```diff
@@ -0,0 +1,551 @@
...
+            "Disabled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "CapabilityStatus",
+            "modelAsString": false
+          }
+        },
+        "reason": {
```
Comment (anuchandy) [inline specification/sql/resource-manager/Microsoft.Sql/2015-05-01/capabilities.json]: Does service populate `reason` field only if `status` has a value other than `Available` (i.e. `Visible`, `Default `, `Disabled `)?

**Comment by anuchandy** [comment]
@ravbhatnagar please sign-off.

**Review by anuchandy — APPROVED** [review]
looks good from SDK side.

File: specification/sql/resource-manager/Microsoft.Sql/2015-05-01/capabilities.json (hunk 1 above)
Comment (gaagra) [reply specification/sql/resource-manager/Microsoft.Sql/2015-05-01/capabilities.json]: yes

File: specification/sql/resource-manager/readme.md [hunk 2]
```diff
@@ -56,6 +55,7 @@ input-file:
 - Microsoft.Sql/2014-04-01/sql.core.json
 - Microsoft.Sql/2014-04-01/usages.json
 - Microsoft.Sql/2015-05-01-preview/blobAuditingPolicies.json
+- Microsoft.Sql/2015-05-01/capabilities.json
```
Comment (jaredmoo) [inline specification/sql/resource-manager/readme.md]: please add 2015-05-01-preview capabilities and use that one here.

**Comment by ravbhatnagar** [comment]
Signed off!

**Comment by salameer** [comment]
@anuchandy 

its seems to me that this is ready for merging right :)

**Pull request merged**